src/modules/commander/Commander.cpp

```diff
@@ -3784,51 +3786,71 @@ void Commander::estimator_check()
 		if (_estimator_selector_status_sub.copy(&estimator_selector_status)) {
 			if (estimator_selector_status.primary_instance != _estimator_status_sub.get_instance()) {
 				_estimator_status_sub.ChangeInstance(estimator_selector_status.primary_instance);
+				_estimator_status_flags_sub.ChangeInstance(estimator_selector_status.primary_instance);
 			}
 		}
 	}
 
-	if (_estimator_status_sub.update()) {
-		const estimator_status_s &estimator_status = _estimator_status_sub.get();
+	if (_estimator_status_flags_sub.update()) {
+		const estimator_status_flags_s &estimator_status_flags = _estimator_status_flags_sub.get();
 
-		// Check for a magnetomer fault and notify the user
-		const bool mag_fault = (estimator_status.control_mode_flags & (1 << estimator_status_s::CS_MAG_FAULT));
-		const bool gnss_heading_fault = (estimator_status.control_mode_flags & (1 << estimator_status_s::CS_GPS_YAW_FAULT));
+		if (estimator_status_flags.cs_inertial_dead_reckoning || estimator_status_flags.cs_wind_dead_reckoning) {
+			_status_flags.position_dead_reckoning = true;
```

Comment:
"Inertial dead reckoning" means: nav solution obtained by previous solution and inertial measurements. Therefore, "Position dead reckoning" sounds a bit strange to me. If in inertial or wind DR, why not calling it simply "dead_reckoning"?

Reply:
In the context of `vehicle_status_flags` I was trying to be more descriptive. I'll update it.